crates/iota/src/iota_commands.rs

```diff
@@ -795,7 +800,7 @@ async fn start(
             .map_err(|_| anyhow!("Invalid indexer host and port"))?;
         tracing::info!("Starting the indexer service at {indexer_address}");
         // Start in writer mode
-        start_test_indexer(
+        start_test_indexer::<diesel::PgConnection>(
```

Comment:
This is weird

Reply:
fixed in https://github.com/iotaledger/iota/pull/4987/commits/43b7ccb110286fa27d5e9fa35e5f5de3404274a3